spmd/compiler/graph_optimization.py

```diff
@@ -166,18 +166,20 @@ def apply(
         return self
 
     @graph_optimization_pass()
-    def fuse_communication(
+    def fuse_communication_ring(
         self,
         bucketing_strategy: BucketingStrategy = BucketingStrategy.FIXED,
         scheduling_policy: SchedulingPolicy = SchedulingPolicy.FCFS,
-        fusion_policy: int = 10,
+        # fusion_policy: int = 2,
```

Comment:
do we need to uncomment this?

Reply:
yes - I had it setup when running so I only needed to mod fusion.py for debugging ease, but we want it to be 'pass through' from SPMD.